tests/common/MonoTouch.Dialog/MonoTouch.Dialog.iOS.csproj

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net$(BundledNETCoreAppTargetFrameworkVersion)-ios;net$(BundledNETCoreAppTargetFrameworkVersion)-maccatalyst;net$(BundledNETCoreAppTargetFrameworkVersion)-macos;net$(BundledNETCoreAppTargetFrameworkVersion)-tvOS</TargetFramework>
```

Comment:
Why all the TargetFrameworks here, but not in the other csprojs?

Reply:
Testing, this file is slated for deletion